### ISSUE TITLE
Project Question API Needs to be updated

### ISSUE THREAD

**Issue by AllenBW** [projectjellyfish/api#767]
Currently project questions returns an `options` array like so:
```
{
  "name": "",
  "help": "",
  "required": false,
  "type": "multiple",
  "options": ["foo", "bar", "baz"]
}
```
We need to have it return an object instead:
```
   "name": "",
   "help": "",
   "required": false,
   "type": "multiple",
   "options": [
      {
         "option": "foo",
         "include": [],
         "exclude": [],
         "load_order": 0
      },
      {
         "option": "bar",
         "include": [],
         "exclude": [],
         "load_order": 1
      }
   ]
}
```

**Comment by AllenBW** [comment]
need to add options to the get! (by following the include pattern of other endpoints)

**Comment by AllenBW** [comment]
gotta beat that sample.rake a bit as well, will do when this is in play

**Comment by OBCENEIKON** [comment]
Currently the `options` returns an array like so:
```
options: ["foo", "bar", "baz"]
```
We need to have it return an object like so:
```
options: [
   {
      "option": "foo",
      "include": [],
      "exclude": [],
      "load_order": 0
   },
   {
      "option": "bar",
      "include": [],
      "exclude": [],
      "load_order": 1
   }
]
```

**Comment by AllenBW** [comment]
and i should mention the include and exclude are tags

**Comment by OBCENEIKON** [comment]
This is ready for PR, waiting on a PR and a merge for #841 first as I had to merge that branch into #767

**Comment by OBCENEIKON** [comment]
It looks like this actually got merged in with #841
